### PR TITLE
added more comments to demo

### PR DESCRIPTION
**Pull request by AndrewBakalinIntel**
As Maria is unavailable, I made all the fixes from https://github.com/opencv/open_model_zoo/pull/240 in current merge request.

### CODE REVIEW

**Comment by kchechil** [comment]
Jenkins please retry a build

File: demos/smart_classroom_demo/README.md [hunk 1]
```diff
@@ -121,11 +121,11 @@ Example of a valid command line to run the application for recognizing first rai
...
 > **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies: 
-* `face-detection-adas-0001`
-* `landmarks-regression-retail-0009`
-* `face-reidentification-retail-0095`
-* `person-detection-action-recognition-0005`
-All other models may work incorrectly on these devices.
+>* `face-detection-adas-0001`
+>* `landmarks-regression-retail-0009`
+>* `face-reidentification-retail-0095`
+>* `person-detection-action-recognition-0005`
```
Comment (Wovchena) [inline demos/smart_classroom_demo/README.md]: I think @IRDonch's comment on alphabetical order should be applied to all such added lists.

Comment (AndrewBakalinIntel) [reply demos/smart_classroom_demo/README.md]: Done

**Comment by IRDonch** [comment]
Jenkins: ok to test

**Pull request merged**